ab3d2_source/Makefile

```diff
@@ -1,34 +1,58 @@
 CC=m68k-amigaos-gcc
+LINK=m68k-amigaos-gcc
 ASS=vasmm68k_mot
-LINK=vlink
+VLINK=vlink
+
 STRIP=m68k-amigaos-strip
 
 #find the toolchain's directory
 PREFIX = $(shell ./getprefix.sh "$(CC)")
 
 #-DCD32VER
 
+CFLAGS = -m68020-60 -mtune=68030 -noixemul -g -ggdb -O3
```

Comment:
Does tuning for 030 offer anything 020 doesn't?

Reply:
In my experience, the -m68020-60 will actually optimize for 060 (like prefer mul over shifts), but avoid instructions that thee 060 doesn't have (like certain 64bit multiplications/divisions). With -mtune 030 I'm aiming for 030 optimization without using 030 instructions the the 060 doesn't have.